BenControlFlow.playground/Contents.swift

```diff
@@ -0,0 +1,245 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+//
+//
+////1
+//var isItSnowing: Bool = true
+//
+//if isItSnowing {
+//
+//    print ("I can do snowboarding ")
+//} else {
+//print ("I can not do snwoboarding")
+//    
+//}
+//
+//var isItBeautiful: Bool = false
+//
+//if isItBeautiful {
+//
+//    print(" I buy a new car")
+//} else {
+//    print(" I keep the old one")
+//
+//}
+//
+//var doYouHaveTimeForBeginners: Bool = true
+//
+//if doYouHaveTimeForBeginners {
+//   
+//    print (" I will be a good programmer")
+//} else {
+//    print (" I will have wasted my time")
+//}
+//
+//
+//var doYouHaveTimeForNewcomers: Bool = false
+//
+//if doYouHaveTimeForNewcomers {
+//   print (" I will be a good programmer")
+//
+//} else {
+//    print (" I will have wasted my time")
+//}
+//
+//
+////2 
+//
+//var temperature = 0
+//if temperature <= 0 {
+//    print(" I am freezing ok!")
+//} else {
+//    print(" I am not freezing")
+//}
+//
+//var isItRaining: Bool = true
+//var temperatureOne = 0
+//
+//if (isItRaining && temperatureOne <= 0) {
+//    print("it is freezing and I am wet")
+//    
+//} else if (isItRaining && temperatureOne > 0) {
+//    print("I am not freezing but i am not wet")
+//
+//} else if (!isItRaining && temperatureOne > 0) {
+//    print("I am not freezing and I am not wet")
+//    
+//} else if (isItRaining && temperatureOne <= 0) {
+//    print ("I am am freezing but i am not wet")
+//}
+//
+////3 and 4
+//
+//let name = "ben"
+//let name1 = "chris" // it cannot change, it is a constant, or Let become a var , or declare a new constant.
+//
+//var age: Int = 35
+//print (age)
+////We have no value for age, it is not initialized yet and we ask the compiler to give an answer that cannot be given.
+//
+//let nameOne = "Chris"
+//var ageOne = 51
+//print (ageOne)
+//
+//
+////5 
+//
+//var middleAged: Bool = true
+//
+//if age >= 35 {
+//    middleAged = true
+//} else {
+//    middleAged = false
+//}
+//
+//if age <= 35 {
+//    middleAged = true
+//} else {
+//    middleAged = false
+//    
+//}
+//
+//if age == 35 {
+//    middleAged = true
+//} else {
+//    middleAged = false
+//}
+//
+//if age != 35 {
+//    middleAged = true
+//} else {
+//    middleAged = false
+//}
+//
+////6
+//
+//var  middleAged1 = age >= 35 ? true: false
+//var  middleAged2 = age <= 35 ? true: false
+//var  middleAged3 = age == 35 ? true: false
+//var  middleAged4 = age != 35 ? true: false
+
```

Comment:
Well done great usage of tenary operators.... like to see you using different operators